ngraph_bridge/ngraph_encapsulate_impl.h

```diff
@@ -44,17 +44,12 @@ class NGraphEncapsulateImpl {
                           std::vector<const Tensor*>& static_input_map,
                           std::stringstream& signature_ss);
 
-  static Status Compile(std::shared_ptr<ngraph::Function> ng_function,
```

Comment:
Are these functions specific to AOT?

Reply:
No, but they were not needed, hence removed.